src/Cell.js

```diff
@@ -229,12 +233,23 @@ var Cell = React.createClass({
       'is-dragged-over-down' :  this.isDraggedOverDownwards(),
       'was-dragged-over' : this.wasDraggedOver()
     });
-    return joinClasses(className, extraClasses, updateCellClass);
+    return joinClasses(className, extraClasses);
   },
 
   getUpdateCellClass() {
-    if(this.state.isRowChanging && this.props.selectedColumn != null){
-      return this.props.column.getUpdateCellClass ? this.props.column.getUpdateCellClass(this.props.selectedColumn, this.props.column, this.state.isCellValueChanging) : '';
+    return this.props.column.getUpdateCellClass ? this.props.column.getUpdateCellClass(this.props.selectedColumn, this.props.column, this.state.isCellValueChanging) : '';
+  },
+
+  applyUpdateClass() {
+    var updateCellClass = this.getUpdateCellClass();
+    // -> removing the class
+    if(updateCellClass != null) {
+      this.getDOMNode().classList.remove(updateCellClass);
+      // -> triggering reflow /* The actual magic */
+      // without this it wouldn't work. Try uncommenting the line and the transition won't be retriggered.
+      this.getDOMNode().offsetWidth = element.offsetWidth;
```

Comment:
or feels like some of teh react animation stuff that was big back in teh day might help? feels like render isnt the right option for something we want to be a temporal flash/highlight?